build.sbt

```diff
@@ -457,11 +457,14 @@ lazy val isScala3 = isScalaBinaryVersion("3")
 def isScala213or3 = Def.setting(isScala213.value || isScala3.value)
 
 lazy val munitLibrary = Def.setting {
+  // TODO: this is getting ridiculous; any way to fix dependency on munit's
```

Comment:
We could use coursier and fetch like in https://github.com/scalameta/metals/blob/main/project/SemanticDbSupport.scala#L18 to generate a map of supported versions :thinking: 

Or I could stop updating Scala 2.13 in munit for the time being if not needed